crates/webidl/src/util.rs

```diff
@@ -559,16 +577,49 @@ impl<'src> FirstPassRecord<'src> {
                     rust_name.push_str(&snake_case_ident(arg_name));
                 }
             }
+            let structural = force_structural || is_structural(signature.orig.attrs.as_ref(), container_attrs);
+            let catch = force_throws || throws(&signature.orig.attrs);
+            let variadic = signature.args.len() == signature.orig.args.len()
+                && signature.orig.args.last().map(|arg| arg.variadic).unwrap_or(false);
+            if variadic {
+                for i in 0..=7 {
+                    ret.extend(self.create_one_function(
+                        name,
+                        &format!("{}_{}", rust_name, i),
```

Comment:
It would mean that names like `time_log_with_label_and_data_3` will become `time_log_with_label_and_data3`. But `3` is not related particularly to `data`, it is just another part of the whole name.